src/composable/orderTypes/Twap.spec.ts

```diff
@@ -129,26 +179,26 @@ describe('Twap', () => {
       isValid: false,
       reason: 'InvalidSpan',
     })
-    expect(Twap.fromData({ ...TWAP_PARAMS_TEST, appData: constants.AddressZero }).isValid()).toEqual({
-      isValid: false,
-      reason: 'InvalidData',
-    })
   })
 
-  test('isValid: Fails if appData has a wrong number of bytes', () => {
+  test('Invalid twap: InvalidData (ABI parse error in appData)', () => {
     // The isValid below test triggers a throw by trying to ABI parse `appData` as a `bytes32` when
     // it only has 20 bytes (ie. an address)
     expect(Twap.fromData({ ...TWAP_PARAMS_TEST, appData: constants.AddressZero }).isValid()).toEqual({
       isValid: false,
       reason: 'InvalidData',
     })
   })
+})
 
+describe('Serialize', () => {
   test('serialize: Serializes correctly', () => {
     const twap = Twap.fromData(TWAP_PARAMS_TEST)
     expect(twap.serialize()).toEqual(TWAP_SERIALIZED(twap.salt))
   })
+})
 
+describe('Deserializes', () => {
```

Comment:
```suggestion
describe('Deserialize', () => {
```

To keep the same verb tense as `Serialize`

Reply:
corrected